smarts/core/local_traffic_provider.py

```diff
@@ -1095,23 +1095,22 @@ def _pick_lane(self, dt: float):
         # Try to find the best among available lanes...
         best_lw = self._lane_windows[my_idx]
 
-        # Check self, then right, then left.
+        # Default current lane then check right lanes then left lanes.
         lanes_to_right = list(range(0, my_idx))[::-1]
         lanes_to_left = list(
-            range(min(my_idx, len(self._lane_windows)), len(self._lane_windows))
-        )
-        cut_in_is_real_lane = self._cutting_into and self._cutting_into.index < len(
-            self._lane_windows
+            range(min(my_idx + 1, len(self._lane_windows)), len(self._lane_windows))
         )
         checks = lanes_to_right + lanes_to_left
-
         # hold lane for some time if added recently
         if self._forward_after_added < self._after_added_hold_secs:
             self._forward_after_added += dt
             # skip checks
             checks = []
 
         ## TODO: Determine how blocked lane changes should be addressed
+        # cut_in_is_real_lane = self._cutting_into and self._cutting_into.index < len(
+        #     self._lane_windows
+        # )
```

Comment:
Just want to confirm this wasn't left commented unintentionally.

Reply:
It is commented out intentionally, this was a consideration of how to address blocking when lanes to right and left are unavailable for periods of time.

Reply:
I should convert this to pseudo-code rather than keeping it in as code.